README.md

```diff
@@ -50,6 +50,8 @@ def user_report
 end
 ```
 
+**Note:** Word sometimes wraps characters in XML tags, causing the replacement to not work. I recommend not using any special characters in your variable names.
```

Comment:
A similar issue, when editing an existing wordfile or copy/pasting in a file, XML tags are sometimes inserted mid variable name.  For example, if you had the variable `<<Client.Contact_Email>>` saved in your doc, and then later went in to edit by selecting Email and replacing with Name, `<<Client.Contact_Name>>`, XML may be inserted surrounding the `Name` portion of the variable.

```
<w:r w:rsidRPr="00C21B05"
>
  <w:rPr>
    <w:rFonts
      w:ascii="Arial"
      w:eastAsia="Times New Roman"
      w:hAnsi="Arial"
      w:cs="Arial" />
    <w:color w:val="312F33" />
    <w:sz
      w:val="20" />
    <w:szCs w:val="20" />
    <w:lang
      w:bidi="en-US" />
  </w:rPr>
  <w:t>&lt;&lt;Client.Contact_</w:t>
</w:r>
<w:r>
  <w:rPr>
    <w:rFonts
      w:ascii="Arial"
      w:eastAsia="Times New Roman"
      w:hAnsi="Arial"
      w:cs="Arial" />
    <w:color w:val="312F33" />
    <w:sz
      w:val="20" />
    <w:szCs w:val="20" />
    <w:lang
      w:bidi="en-US" />
  </w:rPr>
  <w:t>Name</w:t>
</w:r>
```

Currently to get around this, my team has to fully delete the variable name and retype it when this occurs.

Has anyone else encountered this and come up with a solution? Perhaps a way to set a delimiter open/close variables and capture everything in between, and remove any XML occurring between the delimiters?